vars/getJobTimeouts.groovy

```diff
@@ -2,12 +2,13 @@
 
 List<Integer> call(Map params, String region){
     // handle params which can be a json list
+    def current_region = initAwsRegionParam(params.region, region)
```

Comment:
someday we'll need to rename `initAwsRegionParam` to something more suitable...
since it would work also for other backends

@fgelcer did you tried it for GCP/Azure ? i.e. that random works ? (you might want know that before triggering such jobs)

Reply:
i will open a task for that, but the jobs i have are AWS only, so i didn't check other cloud providers

Reply:
to make others working it is needed to update `export SCT_GCE_DATACENTER` and `export SCT_AZURE_REGION_NAME`